### ISSUE TITLE
BUG: Hatchlings do not work as intended

### ISSUE THREAD

**Issue by MustacheMan2** [DarkflameUniverse/DarkflameServer#759]
### Make sure you've done the following:

- [X] I have checked that this issue has not already been reported.

- [X] I have validated that this issue is not a syntax error of either MySQL or SQLite.

- [X] I have pulled the latest version of the main branch of DarkflameServer and have confirmed that the issue exists there.

### DarkflameServer Version

500b788

### Issue Description

These are pet like creatures that are summoned by an object. I don't have much knowledge on this at all. There is no live game evidence of them working and no images, all I have are the in-game objects and a section for them on the wiki. However these were game card rewards, therefore they are finished products semi available to the public during live so I think its a good idea to implement them. I don't know what they do now that is accurate to how they worked in live, but I was able to equip the summoning item, summon the pet, and watch it render in. However they despawn quite quickly and don't have the behaviors for following that the item describes them as having.

### Reproduction Steps

example ids for the packages containing the objects:

14195
14774
14196
14770

### Expected Behavior

The items state that the pets follow you around. The wiki says that they follow the player similar to how regular pets do. That's all I know.

### Environment

ubuntu server

**Comment by EmosewaMC** [comment]
Marking this as enhancement since they were game card rewards and are not mission critical to implement or fix.  Especially considering they are only attainable at the moment with manually adding the item to your character.

**Comment by aronwk-aaron** [comment]
`scripts\02_server\Objects\Hatchlings\L_HATCHLING_PETS.lua`

**Comment by Cromarko** [comment]
Any updates on fix? Hatchlings like 'Brown Dog's Bone', 'White Mouse's Cheese', etc and others don't follow the character when spawned and they disappear.

**Comment by aronwk-aaron** [comment]
no, super low priority, not in the mainline gameplay as they were never officially released
